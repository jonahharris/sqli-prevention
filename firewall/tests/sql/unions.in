>>> Sada testovacich SQL dotazu: union
SELECT 1 + 1 UNION SELECT 2;
SELECT 1 + 1 UNION ALL SELECT 2;
SELECT 1 + 1 UNION DISTINCT SELECT 2;
SELECT * FROM authors WHERE AuthorID = -1 union select 1,2,3,4,5,6 from author;-- 
SELECT * FROM authors WHERE AuthorID = -1 union select 1,concat(authorid,0x3A,email,0x3A,password),3,4,5,6 from authors;-- 
(SELECT e_name AS name, email FROM employees) UNION (SELECT c_name AS name, email FROM customers);
(SELECT 'John Doe' AS name, 'john.doe@example.net' AS email) UNION (SELECT name, email FROM customers);
(SELECT name, email FROM employees) UNION (SELECT name, email FROM customers) ORDER BY name;
(SELECT a FROM t1 WHERE a=10 AND B=1) UNION (SELECT a FROM t2 WHERE a=11 AND B=2) ORDER BY a LIMIT 10;
SELECT 1 UNION SELECT 2 UNION SELECT 3;
SELECT 1 UNION SELECT 2 UNION (SELECT 3) LIMIT 10 OFFSET 25;
