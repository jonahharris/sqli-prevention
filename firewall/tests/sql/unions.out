>>> Sada testovacich SQL dotazu: union
NUMBER 1;NUMBER 1;ADD;SELECTNODATA 0 1;NUMBER 2;SELECTNODATA 0 1;UNION 0;STMT;2;0;
NUMBER 1;NUMBER 1;ADD;SELECTNODATA 0 1;NUMBER 2;SELECTNODATA 0 1;UNION 1;STMT;2;0;
NUMBER 1;NUMBER 1;ADD;SELECTNODATA 0 1;NUMBER 2;SELECTNODATA 0 1;UNION 2;STMT;2;0;
SELECTALL;TABLE authors;NAME AuthorID;NUMBER -1;CMP 4;WHERE;SELECT 0 1 1;NUMBER 1;NUMBER 2;NUMBER 3;NUMBER 4;NUMBER 5;NUMBER 6;TABLE author;SELECT 0 6 1;UNION 0;STMT;2;0;
SELECTALL;TABLE authors;NAME AuthorID;NUMBER -1;CMP 4;WHERE;SELECT 0 1 1;NUMBER 1;NAME authorid;STRING '***';NAME email;STRING '***';NAME password;CALL 5 CONCAT;NUMBER 3;NUMBER 4;NUMBER 5;NUMBER 6;TABLE authors;SELECT 0 6 1;UNION 0;STMT;2;0;
NAME e_name;ALIAS name;NAME email;TABLE employees;SELECT 0 2 1;NAME c_name;ALIAS name;NAME email;TABLE customers;SELECT 0 2 1;UNION 0;STMT;2;0;
STRING '***';ALIAS name;STRING '***';ALIAS email;SELECTNODATA 0 2;NAME name;NAME email;TABLE customers;SELECT 0 2 1;UNION 0;STMT;2;0;
NAME name;NAME email;TABLE employees;SELECT 0 2 1;NAME name;NAME email;TABLE customers;SELECT 0 2 1;NAME name;ORDERBY 0;ORDERBYLIST 1;UNION 0;STMT;2;0;
NAME a;TABLE t1;NAME a;NUMBER 10;CMP 4;NAME B;NUMBER 1;CMP 4;AND;WHERE;SELECT 0 1 1;NAME a;TABLE t2;NAME a;NUMBER 11;CMP 4;NAME B;NUMBER 2;CMP 4;AND;WHERE;SELECT 0 1 1;NAME a;ORDERBY 0;ORDERBYLIST 1;NUMBER 10;LIMIT 1;UNION 0;STMT;2;0;
NUMBER 1;SELECTNODATA 0 1;NUMBER 2;SELECTNODATA 0 1;UNION 0;NUMBER 3;SELECTNODATA 0 1;UNION 0;STMT;3;0;
NUMBER 1;SELECTNODATA 0 1;NUMBER 2;SELECTNODATA 0 1;UNION 0;NUMBER 3;SELECTNODATA 0 1;NUMBER 10;NUMBER 25;LIMIT 3;UNION 0;STMT;3;0;
